netshare/utils/model_configuration.py

```diff
@@ -68,14 +69,23 @@ def _load_config() -> List[Config]:
 
     configs = []
     for config_ in config_post_expand:
-        sub_result_folder = os.path.join(
-            os.path.basename(config_["dataset"]),
-            ",".join(
+        # single-chunk case
+        if "chunkid" not in os.path.basename(config_["dataset"]):
```

Comment:
again, maybe it's worth using `if get_config("global_config.n_chunks", default_value=1) == 1:` which is the cause of the if/else, and not the effect